docs/scanner-pools.md

```diff
@@ -0,0 +1,158 @@
+
+![Add Stake](ApproveStaking.png)
+
+- Enter the amount of *approved* FORT you want to stake and wait for the transaction to confirm
+
+![Add Stake](ApproveFORTStaking.png)
+
+
+### Delegating to a pool
+
+- Find a node pool (that you do not own) on the Network Participants page that has a node and enough stake to earn rewards
+
+![Network Participants](NetworkParticipants.png)
+
+- Navigate to the node pool page by clicking on the address
+
+![Delegated Staking](DelegatedStaking.png)
+
+- Click on the “Delegate” button. There are 2 transactions required for delegation
+    - Approve the amount of FORT to delegate
+    - Delegate the approved FORT
+
+- Enter the amount of FORT you want to approve for delegation. Click “Approve tokens” and approve the transaction. Wait for the transaction to confirm.
+
+![Delegated Staking Approve](DelegatedStakingApprove.png)
+
+- Enter the amount of approved FORT want to delegate and click “Delegate” and approve the transaction. Wait for the transaction to confirm
+
+![Delegated Staking FORT](DSFORT.png)
+
+- Navigate to “My Stakes” or  “Overview” on your profile and you can see your staked FORT
+
+## Creating new nodes
+
+Each node needs to be registered to a specific pool in order to be operational. While one can register many scan nodes on a pool, the pool needs to be populated with more stake first in order to avoid registration issues. This is for preventing pool shutdown conditions that occur when the average per-scanner stake is below minimum.
+
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on the node pool you would like to add a node
+![My Node Pools List](MyNodePoolsList.png)
+
+- Make sure you have at least 2,500 FORT staked for the new node. Click “Add Scan Node”. Use the Forta node CLI to generate the scanner authorization token (described [here](scanner-quickstart#register-scan-node)) needed to claim a node into a pool.
+
+![Add Scan Node](AddScanNode.png)
+
+## Disabling existing nodes
+
+If a node is no longer used, it should be disabled to free up allocated stake for newer nodes.
+
+- Navigate to the node pool that has the node you want to disable
+
+- Select the disable action on the node you want to disable
+
+![Disable Scan Node](DisableScanNode.png)
+## Adjusting stake allocation
+
+While it's always possible to add more stake to a pool, it is allocated proportional to the amount of scan nodes in the pool. Total allocated stake can only be `node_count × max_stake_per_scan_node`.
+
+For example, if there are two scan nodes in a pool, a total of 25k FORT deposited and only 5k FORT worth of stake is allowed per scan node, then the pool can allocate a maximum of `5k × 2 = 10k` FORT in total. This is the amount that is effective in calculating the rewards and the rest of `25k - 10k = 15k` remains unallocated until more scan nodes are registered to the pool.
```

Comment:
is the max stake per scan node mentioned anywhere? is 5k the actual number?

Reply:
Mentioning this here now - fixing.